roles/openshift_health_checker/library/aos_version.py

```diff
@@ -78,7 +82,7 @@ def bail(error):  # pylint: disable=missing-docstring
         for name in not_found:
             msg += '  %s\n' % name
         bail(msg + 'Please check your subscriptions and enabled repositories.')
-    if multi_found:
+    if multi_found and rpm_prefix == 'atomic-openshift':
```

Comment:
I don't really want to disable the whole check, just this part of it. I agree checking rpm prefix is a bit weird... will see if I can clean this up.